e2e/fee_middleware_test.go

```diff
@@ -2,6 +2,8 @@ package e2e
 
 import (
 	"context"
+	"e2e/testsuite"
```

Comment:
Sorry to keep bothering you with the imports, but do you mind to put these ones back where they were before?